wrappers/kn/src/androidNativeTest/kotlin/Test.androidNative.kt

```diff
@@ -0,0 +1,4 @@
+import zxingcpp.ImageView
```

Comment:
In Kotlin native, for different target, there can be different source-sets, this file is placed at `wrappers/kn/src/androidNativeTest/kotlin/Test.androidNative.kt`, indecating it belongs to androidNativeTest source set.
According to https://kotlinlang.org/docs/multiplatform-hierarchy.html#additional-configuration, there's hierarchy relationship between sourcesets, the higher level source sets will define interface by `expect` keyword, and the lower leved source sets have to implement them with `actual` keyword. 
When it's compiling, the sourcesets will be put together to compile, so identical file name will cause conflicts(as what i remember, and this is also default naming method from ide).

Reply:
I see. But since this file is basically empty and there is currently no CI code that would actually execute this, I'd rather not add this at this time.

Reply:
Umm, but it's kotlin that it requires every source sets must have an corresponding implementation

Reply:
So, at least there must be a "stub" function for pass compilnation

Reply:
If you don't have any further questions, I‘m going to mark this as resolved :)

Reply:
Ok, so there can be multiple source sets but that seems not be a strict requirement but rather a configuration option. So why not disabling it then, if there are no platform specific parts involved?

Reply:
Now, the circumstance is that the main source set as well as binding itself works well on all the targets(including androidNative linux mingw apple), but the decoding library used in testing is not available for androidNative target...

I don't think create another binding on stb_image again is a good idea, because in that case we(or me)'ll have to publish two library just for one binding, it's too "heavy".

Reply:
> Why does the string "android" come up in the file name here?

And it should be noted that androidNative is not a traditional android jvm target, but a native target.
Maybe you misunderstand this.

Reply:
And as far as I know, kotlin can only disable the whole building target, which means we cannot disable test source-set alone with out disabling main source-set

Reply:
The stb_image now we're using is from an external library called `korim`(awa the decoding library used before) which doesn't support androidNative targets.

Reply:
Yes, we finally get rid of that, thanks for you inspiration: https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/858318ff180dd5ee11e06c11fff77faae1bc7930